verify.go

```diff
@@ -276,6 +279,32 @@ func (v *IDTokenVerifier) Verify(ctx context.Context, rawIDToken string) (*IDTok
 		}
 	}
 
+	// If nbf claim is provided in token, ensure that it is indeed in the past.
+	if !v.config.SkipNbfCheck {
+		var claimsMap map[string]interface{}
+		err := json.Unmarshal(t.claims, &claimsMap)
+		if err != nil {
+			return nil, fmt.Errorf("oidc: failed to unmarshal claims: %v", err)
+		}
+		if nbfRaw := claimsMap["nbf"]; nbfRaw != nil {
+			nbf, ok := nbfRaw.(float64)
+			if !ok {
+				return nil, fmt.Errorf("oidc: invalid nbf claim, not a number: %v", err)
+			}
+			nbfTime := time.Unix(int64(nbf), 0)
+
+			now := time.Now
+			if v.config.Now != nil {
+				now = v.config.Now
+			}
+			nowTime := now()
```

Comment:
If we go with SkipExpiryCheck, move:

https://github.com/coreos/go-oidc/blob/274971e2c94cd6fb17614979e2899edbc81af146/verify.go#L269-L272

Up one block and reuse now so both checks work of the same time.